CHANGELOG.md

```diff
@@ -2,8 +2,6 @@
 
 This changelog keeps track of all changes to the Packs SDK. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
```

Comment:
Why remove this?

Reply:
I was wrestling with `make release-manual` - thought maybe I could bypass release notes for this release since there are no external-facing updates.

Reply:
in the future, i think it'd be better to do releases as separate PRs from the actual changes